src/mesido/esdl/asset_to_component_base.py

```diff
@@ -323,6 +327,84 @@ def _pipe_get_diameter_and_insulation(self, asset: Asset) -> Tuple[float, list,
 
         return inner_diameter, insulation_thicknesses, conductivies_insulation
 
+    def _gas_pipe_get_diameter_and_roughness(self, asset: Asset) -> Tuple[float, float]:
+        """
```

Comment:
The word "insulation" should be removed in all the comment as well as the in the logger.warning(...)

Reply:
Yes this check has even been removed, unnecessary for gas pipes

Reply:
I see insulation still appears in the "Returns: .... -->> and conductivity of each insulation layer"

Reply:
My bad, I overlooked that one